tests/test_glob.py

```diff
@@ -115,6 +115,7 @@ def test_recursive(self):
         self._test_matches(
             'Foo/**',
             [
+                'Packages/Foo/',
```

Comment:
Should we match this at all?

Reply:
Yes, because a UNIX glob would and I don't think we need to deviate. It's a corner case that probably won't come up very often, but if it does then I think handling it like a UNIX glob is the least surprising thing to do.